eng/scripts/Publish-MavenPackages.ps1

```diff
@@ -187,13 +187,18 @@ Write-Information "Repository Password is: [redacted]"
 Write-Information "GPG Executable Path is: $GPGExecutablePath"
 Write-Information "Group ID Filter is: $GroupIDFilter"
 Write-Information "Artifact ID Filter is: $ArtifactIDFilter"
+Write-Information "Stage Only is: $StageOnly"
```

Comment:
What about snapshot support? I know a number of partners rely on publishing those.

Reply:
I'll probably cover this with a separate PR (we can run both pipelines side by side for a bit). The snapshot publishing process is quite a bit different and will take more time to validate, and I want to make sure this pipeline is ready for the team that needs it for Android publishing.